### PR TITLE
Add documentation on operator test playbooks CI

### PR DESCRIPTION
**Pull request by sonam1412**

### CODE REVIEW

**Review by jsztuka — APPROVED** [review]
LGTM

File: docs/operatorTestPlaybooksCI.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg).
```
Comment (nmars) [inline docs/operatorTestPlaybooksCI.md:9]: I suggest adding: `For more information on github2fedmsg and Fedora Messaging see https://github.com/fedora-infra/github2fedmsg.`

File: docs/operatorTestPlaybooksCI.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Purpose
+This document is to describe about the working of the Upstream Operator Test Playbooks CI Pipeline.
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:2]: Try this: `This document describes the Upstream Operator Test Playbooks CI Pipeline and how it works`

File: docs/operatorTestPlaybooksCI.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
+
+* The job is triggered by the Fedora Messaging messages which:
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:14]: `...by a Fedora Messaging message which` or `by a message on the Fedora Messaging bus`

File: docs/operatorTestPlaybooksCI.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+  * Indicate that a pull request was opened, re-opened, or synchronized (“synchronized” meaning that code changes were pushed to an already-opened pull request)
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:16]: `Indicates`

File: docs/operatorTestPlaybooksCI.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+    Ex: [PR-opened](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.pull_request.opened)
+
+  * Indicate a comment was added that contains the text `/retest` or `/test CVP`
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:20]: `Indicates`

File: docs/operatorTestPlaybooksCI.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+    Ex: [PR-comment-added](https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.github.issue.comment&delta=3600)
+
+* The openshift project for the `test-operator-test-playbooks` is `cvp-ci-operatortestplaybooks`
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:24]: You can remove the second `the`: `The openshift project for test-operator-test-playbooks...`

File: docs/operatorTestPlaybooksCI.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+* The test-operator-test-playbooks job parses the fedmsg to determine what files have changed and what repository and branch need to be tested.
+
+* If the data can be successfully parsed, test-operator-test-playbooks uses `CVP bot account` on GitHub to create a check status, `CVP/pr-sanity-test`, for the pull request.
+
+* Note: If the only file that has changed is README.md, test-operator-test-playbooks will then exit successfully. No further tests are required since no ansible playbooks have changed.
+
+* The test-operator-test-playbooks job next configures it’s own [Team Jenkins server](https://jenkins-cvp-ci-operatortestplaybooks.cloud.paas.psi.redhat.com/) to run the testing pipelines.
```
Comment (14rcole) [inline docs/operatorTestPlaybooksCI.md:32]: Move the `next` to the beginning of the sentence

**Review by 14rcole — CHANGES_REQUESTED** [review]

File: docs/operatorTestPlaybooksCI.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+# Introduction
+The Operator Test Playbooks CI Pipeline exists to automatically run tests against pull requests in the redhat-operator-ecosystem/operator-test-playbooks GitHub repository.
+
+# Configuration
+A webhook is configured so that the repository sends events to the github2fedmsg web application.
+Any activity (ex: adding comments) in the repository is therefore sent to github2fedmsg, which then rebroadcasts on the Fedora Messaging bus (fedmsg). 
+
+# Usage
+* The [test-operator-test-playbooks](https://jenkins-cvp-ci.cloud.paas.psi.redhat.com/job/test-operator-test-playbooks/) job runs on the CVP CI Jenkins server.
```
Comment (lslebodn) [inline docs/operatorTestPlaybooksCI.md:12]: There should not be any internal links in upstream documentation.
It is unfair for the community.

File: docs/operatorTestPlaybooksCI.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+* The test-operator-test-playbooks job next configures it’s own [Team Jenkins server](https://jenkins-cvp-ci-operatortestplaybooks.cloud.paas.psi.redhat.com/) to run the testing pipelines.
+
+* When this is complete the test-operator-test-playbooks currently triggers three test pipelines on it’s team Jenkins server:
+
+  * cvp-redhat-operator-bundle-image-validation-test
+  * cvp-isv-operator-metadata-validation-test
+  * cvp-redhat-operator-metadata-validation-test
+
+* These are the same pipelines that run in production. The operators used for testing in these pipelines are the same used in [CVP-CI setup tests](https://gitlab.sat.engineering.redhat.com/cvp/pipeline/-/tree/master/ci/tests/setupTests/sample-ci-messages).
```
Comment (lslebodn) [inline docs/operatorTestPlaybooksCI.md:40]: There should not be any internal links in upstream documentation.
It is unfair for the community.

File: docs/operatorTestPlaybooksCI.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+* The pipelines are running in parallel, although they are started one minute apart from each other. 
+
+* When a testing pipeline is started, the test-operator-test-playbooks job adds a new check status to the pull request on GitHub indicating the name of the test pipeline and it sets the status to “pending”, so that end-users are informed that a test is in progress.
+
+* When a testing pipeline completes, the check status is updated with either `success` or `failure`.
+
+* Each pipeline’s artifacts are uploaded to Amazon S3, the same artifacts as are uploaded in production, and can be viewed externally to determine which stages in the testing pipeline failed. 
+
+  * [Example coldstorage URL](http://external-ci-coldstorage.datahub.redhat.com/cvp/cvp-redhat-operator-bundle-image-validation-test/e2e-test-operator-container-8.0-367/2b303e1a-a4a9-4815-b604-44d67c81dc64/)
```
Comment (lslebodn) [inline docs/operatorTestPlaybooksCI.md:50]: There should not be any internal links in upstream documentation.
It is unfair for the community.

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**